plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/StaticMetastoreConfig.java

```diff
@@ -28,11 +28,14 @@
 public class StaticMetastoreConfig
 {
     public static final String HIVE_METASTORE_USERNAME = "hive.metastore.username";
+    public static final String HIVE_METASTORE_CATALOGNAME = "hive.metastore.catalog.name";
```

Comment:
```suggestion
public static final String HIVE_METASTORE_CATALOG_NAME = "hive.metastore.catalog.name";
```

Reply:
also `hive.metastore.catalog-name`

Reply:
not sure,

but for other constant to config-name it is like
`public static final String HIVE_METASTORE_USERNAME = "hive.metastore.username";` 

which is `_` is replaced with `.` 

so, based on that should we keep it like this only?

`HIVE_METASTORE_CATALOG_NAME = "hive.metastore.catalog.name";`

Reply:
@anusudarsan you still want to change `hive.metastore.catalog.name` to `hive.metastore.catalog-name`?